containers/src/Kokkos_OffsetView.hpp

```diff
@@ -493,7 +493,7 @@ class OffsetView : public ViewTraits<DataType, Properties...> {
       return m_map.m_impl_handle[j0 * m_map.m_impl_offset.m_stride.S0 +
                                  j1 * m_map.m_impl_offset.m_stride.S1];
     }
-#if defined KOKKOS_COMPILER_INTEL
+#if defined(KOKKOS_COMPILER_INTEL)
```

Comment:
Is `(defined(KOKKOS_COMPILER_NVCC) && KOKKOS_COMPILER_NVCC >= 1130)` not needed for this line?

Reply:
No, this location didn't trigger the warning, and it is within a kernel (I tested with the added guard and it warned about `__builtin_unreachable()` lacking device markings)